native/packages/react-native-bpk-component-image/stories.js

```diff
@@ -58,6 +59,7 @@ storiesOf('react-native-bpk-component-image', module)
   ))
   .add('With Loading Behaviour', () => (
     <BpkImageWithLoading
+      imageComponent={CustomImage}
```

Comment:
I think this test should remain the same - this functionality is now tested on line 91.